### PR TITLE
I have added image class in css file by replacing global css

### LINKED ISSUE #102: Create a new class for the img and replace it with the global css in the styles.css file
Create a new class for the img and replace it with the global css in the styles.css file

### PR DESCRIPTION
**Pull request by codesmith25103**
## Purpose

The purpose of this Pull Request is to fix #102

## Goals

To add class css  in css file by replacing global css of img in css file

##  Checklist
- [yes] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ yes] I have read and understood the development best practices guidelines
- [ yes] My code follows the style guidelines of this project
- [yes ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -10,6 +10,15 @@
 }
 
 .title {
+<<<<<<< HEAD
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:13]: Remove these unwanted codes in every file @codesmith25103

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -105,29 +105,26 @@
...
+
       </p>
     </center>
 
     <center>
-<<<<<<< HEAD
+
       <img class="img" src="assets/img/CALCULATOR.png" />
-=======
       <img class="img" src="assets/img/CALCULATOR.png" />
```
Comment (YoshithaRathnayake) [inline index.html:115]: Two images? @codesmith25103 can you close this pr and do the changes in another pr because some unwanted commits have been added to this pr ok?

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (codesmith25103) [reply index.html:115]: ok.. i will add another pr

Comment (YoshithaRathnayake) [reply index.html:115]: Shall I close this pr @codesmith25103 is it ok then
